frontend/package.json

```diff
@@ -10,9 +10,11 @@
     "fix": "prettier --write . && eslint --fix --ext .jsx ."
   },
   "dependencies": {
+    "axios": "^1.1.3",
     "prop-types": "^15.8.1",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-player": "^2.11.0",
```

Comment:
I didn't find it's use on your project. Should it be deleted ?

Reply:
yes, it was just a try to use it. May we keep it as Valentin know he's going to need it after?

Reply:
I'd prefer you to delete it and Valentin can install it as needed

Reply:
ok done! i've just merged it with the branch which has just been added to dev